pallets/asset/src/lib.rs

```diff
@@ -1404,12 +1408,16 @@ impl<T: Config> Module<T> {
             weight_meter,
         )?;
 
-        Self::deposit_event(RawEvent::Transfer(
-            from_portfolio.did,
+        Self::deposit_event(RawEvent::AssetBalanceUpdated(
+            to_portfolio.did,
```

Comment:
Why was this DID changed?  Also I think we should have been using the `Identity::CurrentDid` or pass the caller_did in to save doing a read on each asset transfer (settlement leg).